win32ss/user/ntuser/csr.c

```diff
@@ -63,6 +63,20 @@ ResetCsrApiPort(VOID)
     CsrApiPort = NULL;
 }
 
+BOOL
+InitializeGreCSRSS(VOID)
+{
+    /* Initialize DirectX graphics driver */
+    if (DxDdStartupDxGraphics(0, NULL, 0, NULL, NULL, gpepCSRSS) != STATUS_SUCCESS)
+        return FALSE;
+
+    // FIXME: it's not the full implementation of the function.
+    //        We should call UserGetLanguageID() and sets its data.
+    //        Also enable fonts drivers, initialize font table & stock fonts.
```

Comment:
Currently, `UserGetLanguageID()` and the font support is initialized at the end of our win32k's `DriverEntry`, see https://git.reactos.org/?p=reactos.git;a=blob;f=win32ss/user/ntuser/main.c;hb=568383c9b91c2e529ae4a3609e416ec8044cfc23#l1053

Reply:
In Windows there's still some global GRE initialization that happens in DriverEntry() (and not in the NtUserInitialize); you have to check what it is, though...

Reply:
Also additional note: I did not see any problems/regressions after my latest changes, at least in VMWare.